cypress/e2e/50-logs.cy.ts

```diff
@@ -1,4 +1,56 @@
+import * as logs from '../composables/logs';
+import * as ndv from '../composables/ndv';
+import * as workflow from '../composables/workflow';
+import Workflow from '../fixtures/Workflow_if.json';
+
 describe('Logs', () => {
-	// TODO: the test can be written without AI nodes once https://linear.app/n8n/issue/SUG-39 is implemented
-	it('should open NDV with the run index that corresponds to clicked log entry');
+	it('should show input and output data of correct run index and branch', () => {
```

Comment:
looking forward to seeing more tests here. Created a ticket like we discussed before https://linear.app/n8n/issue/SUG-47/tech-debt-write-tests-for-new-logs-view